pkg/swarm/test_helpers_test.go

```diff
@@ -1,21 +1,34 @@
-// Copyright 2020 The Swarm Authors. All rights reserved.
+// Copyright 2023 The Swarm Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package test_test
+package swarm_test
 
 import (
 	"encoding/binary"
 	"testing"
 
 	"github.com/ethersphere/bee/pkg/swarm"
-	"github.com/ethersphere/bee/pkg/swarm/test"
 )
 
-// TestRandomAddressAt checks that RandomAddressAt generates a correct random address
+func Test_RandAddress(t *testing.T) {
```

Comment:
Maybe try to employ property based testing: https://pkg.go.dev/testing/quick

Reply:
I am not quite sure how should that work. Can we move forward with this PR and have those improvements in another PR? Could you open that PR (it might be easier to see what you had in mind that way)?